_extras/design.md

```diff
@@ -118,10 +118,84 @@ How do I...
 ### Variables and Assignment (9:15)
 
 *   Teaching: 10 min
+
+Basic assignment of a value.  What kind of thing is it?  Can assign results
+of computation and use a variable on the right to assign to the left.
+
+```
+a = 3
+print(a)
+type(a)
```

Comment:
I've never had anyone balk at `print`, or even at `print(type(x))`, but 
you're right, for present purposes we can just use `type(x)`, since 
we're speaking to fellow instructors for the moment, not to learners - 
sorry for having picked nits.